lib/assets/javascripts/utils/ariatiseForm.js

```diff
@@ -77,6 +77,8 @@ const getValidationTypeForElement = (el) => {
       return $(el).attr('type'); // available types at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_<input>_types
     } else if ($(el).is('select')) {
       return 'select';
+    } else if ($(el).is('.tinymce')) {
```

Comment:
we have data-validation attribute for handling cases that are not wise enough with just type's element. Moreover, using the class tinymce to validate that element assumes that we have to define that class any time a textarea is tinymce augmented. There are textareas in other views that do not define tinymce class to load tinymce and therefore ariatising the form will not work for those textareas.